src/zenml/cli/__init__.py

```diff
@@ -741,7 +741,7 @@
 -----------------------------------------
 
 Model deployers are stack components responsible for online model serving.
-They are responsible for deploying models to a remote server, model deployers
+They are responsible for deploying models to a remote server. model deployers
```

Comment:
```suggestion
They are responsible for deploying models to a remote server. Model deployers
```

Sorry! I forgot to make this one a capital M!